pkg/syncer/util.go

```diff
@@ -95,3 +96,28 @@ func getQueryResults(ctx context.Context, volumeIds []cnstypes.CnsVolumeId, clus
 	}
 	return allQueryResults, nil
 }
+
+// GetVolumeID returns volume handle if successfully found by the volumeMigrationService
+func GetVolumeID(ctx context.Context, pv *v1.PersistentVolume) (string, error) {
+	log := logger.GetLogger(ctx)
+	volumeHandle, err := volumeMigrationService.GetVolumeID(ctx, pv.Spec.VsphereVolume.VolumePath)
+	if err != nil {
+		log.Errorf("failed to get VolumeID from volumeMigrationService for volumePath: %q", pv.Spec.VsphereVolume.VolumePath)
+		return "", err
+	}
+	return volumeHandle, nil
+}
+
+// HasMigratedToAnnotation returns true if the migrated-to annotation is found in the newer object
+func HasMigratedToAnnotation(ctx context.Context, prevAnnotations map[string]string, newAnnotations map[string]string) bool {
```

Comment:
Can we change this to `IsMigratedToAnnotationAdded` as we are comparing older annotations with newer annotations?

Debug message "migrated-to annotation not found", will create ambiguty later.

Reply:
Done